tests/test_lazy_json_backends.py

```diff
@@ -506,3 +510,75 @@ def test_lazy_json_backends_hashmap(tmpdir):
         assert get_all_keys_for_hashmap("lazy_json") == ["blah"]
         remove_key_for_hashmap("lazy_json", "blah")
         assert get_all_keys_for_hashmap("lazy_json") == []
+
+
+class TestGithubLazyJsonBackend:
+    base_url = "https://github.com/lorem/ipsum/raw/master"
+
```

Comment:
Given that we're reading from a repo that should be there, we should run these tests against the actual graph github repo instead of using mocks.

Reply:
We now do that, except in those cases where it's not possible (we cannot guarantee a JSON file to not change its content in the actual graph)